--- origsrc/ortp-0.13.1/src/Makefile.am	2006-06-25 20:36:28.000000000 -0500
+++ src/ortp-0.13.1/src/Makefile.am	2008-01-27 17:35:23.250000000 -0600
@@ -32,7 +32,7 @@
 
 libortp_la_LIBADD= $(PTHREAD_LIBS) -lm
 
-libortp_la_LDFLAGS= -version-info $(LIBORTP_SO_VERSION)
+libortp_la_LDFLAGS= -version-info $(LIBORTP_SO_VERSION) -no-undefined
 
 
 SUBDIRS= . tests
--- origsrc/ortp-0.13.1/src/rtpsession_inet.c	2007-03-22 15:06:05.000000000 -0500
+++ src/ortp-0.13.1/src/rtpsession_inet.c	2008-01-27 17:41:22.953125000 -0600
@@ -107,6 +107,7 @@
 			    }
 			}
 		      break;
+#ifdef ORTP_INET6
 		    case AF_INET6:
 		      if (IN6_IS_ADDR_MULTICAST(&(((struct sockaddr_in6 *) res->ai_addr)->sin6_addr)))
 			{
@@ -123,6 +124,7 @@
 			    }
 			}
 		      break;
+#endif /*ORTP_INET6*/
 		  }
 #endif /*hpux*/
 		break;
@@ -288,6 +290,7 @@
 
  		} break;
 
+#ifdef ORTP_INET6
         case AF_INET6: {
 
 			retval= setsockopt(session->rtp.socket, IPPROTO_IPV6, IPV6_MULTICAST_HOPS, 
@@ -299,6 +302,7 @@
 					 (SOCKET_OPTION_VALUE) &session->multicast_ttl, sizeof(session->multicast_ttl));
 
         } break;
+#endif
 
         default:
             retval=-1;
@@ -364,6 +368,7 @@
 
  		} break;
 
+#ifdef ORTP_INET6
         case AF_INET6: {
 
 			retval= setsockopt(session->rtp.socket, IPPROTO_IPV6, IPV6_MULTICAST_LOOP, 
@@ -375,6 +380,7 @@
 				 (SOCKET_OPTION_VALUE)	&session->multicast_loopback, sizeof(session->multicast_loopback));
 
         } break;
+#endif
 
         default:
             retval=-1;
